code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -53,36 +53,39 @@
 		/datum/reagent/oxygen,
 		/datum/reagent/phosphorus,
 		/datum/reagent/potassium,
-		/datum/reagent/uranium/radium,
 		/datum/reagent/silicon,
 		/datum/reagent/silver,
 		/datum/reagent/sodium,
+		/datum/reagent/iron,
+		/datum/reagent/water,
+		/datum/reagent/oil,
+		/datum/reagent/uranium/radium,
 		/datum/reagent/stable_plasma,
-		/datum/reagent/consumable/sugar,
 		/datum/reagent/sulfur,
-		/datum/reagent/toxin/acid
 	)
 	var/list/t2_upgrade_reagents = list(
-		/datum/reagent/iron,
-		/datum/reagent/water,
-		/datum/reagent/fuel
+		/datum/reagent/consumable/sugar,
```

Comment:
why sugar specifically it's used in like 3 things of value

Reply:
It was something as a trade-off for giving chemists Iron and Water back, same with Acid